src/test/java/uk/gov/ons/ctp/response/casesvc/PubSubTestEmulator.java

```diff
@@ -0,0 +1,125 @@
+package uk.gov.ons.ctp.response.casesvc;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import com.google.api.core.ApiFuture;
+import com.google.api.gax.core.CredentialsProvider;
+import com.google.api.gax.core.NoCredentialsProvider;
+import com.google.api.gax.grpc.GrpcTransportChannel;
+import com.google.api.gax.rpc.FixedTransportChannelProvider;
+import com.google.api.gax.rpc.TransportChannelProvider;
+import com.google.cloud.pubsub.v1.*;
+import com.google.cloud.pubsub.v1.stub.GrpcSubscriberStub;
+import com.google.cloud.pubsub.v1.stub.SubscriberStubSettings;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.*;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+/** * This is a PubSub Emulator class which is used for testing pubsub function */
+public class PubSubTestEmulator {
```

Comment:
This class was in originally in the run-time packages. Its been moved here to the test packages as that's what it's purpose/config is.